graylog2-server/src/main/java/org/graylog2/security/ldap/LdapConnector.java

```diff
@@ -251,6 +253,12 @@ public LdapEntry search(LdapNetworkConnection connection,
                     }
                 }
             }
+        } catch (Exception e) {
+            LOG.warn(
```

Comment:
I'm not so sure about the system notification, because this gets triggered every time someone logs in.
People will notice really quickly that something is wrong, because group mapping is not working for any user.